assets/scss/licenses.scss

```diff
@@ -45,3 +45,41 @@
     content: '\00a9';
   }
 }
+
+figcaption {
+  background: $white;
+  border-top-left-radius: $border-radius-small;
+  border-top-right-radius: $border-radius-small;
+  bottom: 0;
+  font-size: 0.625rem;
+  left: 1rem;
+  line-height: 1;
+  padding: 0.5rem;
+
+  &.is-external-link:after {
```

Comment:
I'm not seeing this icon, I think it may be wrapping onto the next line.

Reply:
Fixed in ab1950d